invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -100,10 +101,21 @@ export class RecordCitationField extends Component {
       };
     });
 
+    // convert links in text to clickable links (ignoring punctuations at the end)
+    const escapedCitation = _escape(citation); // escape html characters
+    const urlRegex = /(https?:\/\/[^\s.,;]+)/g;
```

Comment:
This seems to stop too early. `const urlRegex = /(https?:\/\/[^\s,;]+(?=[^\s.,;]*\S))/g;` seems to do the trick.

Reply:
Thanks, I think I've committed the wrong iteration after playing at https://regex101.com/ 😅